dbms/src/TIDB/Codec/Datum.cpp

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include <Storages/Transaction/Datum.h>
+#include <TIDB/Codec/Datum.h>
```

Comment:
"TiDB" is more appropriate than "TIDB" since "Ti" stands for Titanium orz. Please rename the directory to TiDB

ditto as https://github.com/pingcap/tiflash/pull/4932#discussion_r878078340

Reply:
done